### DISCUSSION TITLE
Trending links don't show up in explore tab

### DISCUSSION THREAD

**Discussion by sphinxc0re** [mastodon/mastodon#18366] [Q&A]
I recently moved my instance to the latest Mastodon version (3.5.2). I already approved a bunch of authors but nothing shows up in the "news" section of the explore tab. Am I missing something?

EDIT: I checked the console and there is no error and the api really returns an empty array.

EDIT2: Trends are enabled and I don't have any problems with trending hashtags or posts.

**Comment by ClearlyClaire** [comment]
Are trends enabled? `/admin/settings/edit#form_admin_settings_trends`

**Reply by sphinxc0re** [reply]
Of course! Trending posts do show up as well as trending hashtags

**Reply by ClearlyClaire** [reply]
It is possible that no link have been actually trending yet? Only links to articles (that is, provide enough metadata to be considered as such) are recorded, and only if posted with public visibility and not under a Content Warning. All trends are also localized, so it is possible that if links are indeed trending, none are trending for your specific locale.

**Reply by sphinxc0re** [reply]
It seems like you're correct. When I change my interface language to German, I can see all the articles. Couldn't you use the enabled languages in the `/settings/preferences/other` route instead of the interface language?

**Reply by Gargron** [reply]
The trends system does not currently support showing more than one language. That setting is also not very widely known.
